jetstream/core/orchestrator.py

```diff
@@ -376,13 +387,25 @@ def __init__(
         )
         for idx in range(len(self._generate_engines))
     ]
-    self.detokenize_threads = [
+    self.prefill_detokenize_threads = [
+        JetThread(
+            target=functools.partial(
+                self._detokenize_thread,
+                True,
+                idx,
+            ),
+            name=f"prefill_detokenize-{idx}",
+        )
+        for idx in range(len(self._generate_engines))
+    ]
+    self.generate_detokenize_threads = [
         JetThread(
             target=functools.partial(
                 self._detokenize_thread,
+                False,
```

Comment:
Nit: add keyword `is_prefill` when creating the threads since it's not obvious what True/False correspond to at a glance.

Reply:
Thanks for the advice! Done.